### PR TITLE
feat: zimbraAuthFallbackToLocal is by default set to TRUE

### PR DESCRIPTION
**Pull request by Polpetta**
* Regenerate attrs.xml based codebase

[CO-338]

### CODE REVIEW

**Review by frisonisland — APPROVED** [review]

File: store/conf/attrs/attrs.xml [hunk 1]
```diff
@@ -1470,7 +1470,8 @@ TODO - add support for multi-line values in globalConfigValue and defaultCOSValu
   <desc>explicit mapping to an external LDAP dn for a given account</desc>
 </attr>
 
-<attr id="257" name="zimbraAuthFallbackToLocal" type="boolean" cardinality="single" optionalIn="domain">
+<attr id="257" name="zimbraAuthFallbackToLocal" type="boolean" cardinality="single" optionalIn="globalConfig" flags="domainInherited">
```
Comment (keshavbhatt) [inline store/conf/attrs/attrs.xml:1473]: We also need to make it optionalIn Domain. 
So, it will go like this `optionalIn="globalConfig,domain"`.

**Review by keshavbhatt — CHANGES_REQUESTED** [review]
Dropped a comment to fix the build issue.

Since the changes to attribute will also affect the globalCongig and Server entry I also suggest to run the code generator in store module `ant -Dcarbonio.buildinfo.version=22.8.0_ZEXTRAS_202208 generate-getters` in the store module to bring in the new changes.

File: store/conf/attrs/attrs.xml (hunk 1 above)
Comment (keshavbhatt) [reply store/conf/attrs/attrs.xml:1473]: @frisonisland It happens, no worries :)

Comment (Polpetta) [reply store/conf/attrs/attrs.xml]: Updated!

**Review by keshavbhatt — APPROVED** [review]

**Pull request merged**